### PR TITLE
update 2.10.0

### PR DESCRIPTION
**Pull request by Arishamays1**
## Version Bump to 2.10.0
- updated checksum and version number
- more up to date about section

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -45,8 +44,12 @@ about:
   home: https://pycodestyle.readthedocs.io/
   license: MIT
   license_file: LICENSE
+  license _url: https://github.com/PyCQA/pycodestyle/blob/main/LICENSE
```
Comment (ELundby45) [inline recipe/meta.yaml:47]: There is an extra space in `license _url:`: However, we should remove `license_url` and `doc_source_url` if `license_file` and `doc_url` are present per the updated recipe guidelines.

**Review by ELundby45 — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (Arishamays1) [reply recipe/meta.yaml:47]: Got it! Made the necessary change.

**Review by ELundby45 — APPROVED** [review]
LGTM

**Review by varlackc — COMMENTED** [review]
The `doc_url` seems to have change 

from 
https://pycodestyle.readthedocs.io/

to
https://pycodestyle.pycqa.org

This point might be up to debate, but the upstream still seems to support older versions of python.
https://github.com/PyCQA/pycodestyle/blob/2.10.0/setup.py#L34

**Comment by Arishamays1** [comment]
> https://github.com/PyCQA/pycodestyle/blob/2.10.0/setup.py#L34

Thank you! I made the change regarding the doc_url. 
Considering that we do not support python versions below 3.7, is it necessary to include the use of 3.6?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -28,14 +27,14 @@ requirements:
...
-    - python >=3.6
+    - python
 
 test:
   imports:
     - pycodestyle
   requires:
     - pip
-    - python <3.10
+    - python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:37]: python can be removed from test/requires. It's already in `run`

**Review by varlackc — COMMENTED** [review]
Other than the issue with `python` in the `test section`, everything else looks good.
https://github.com/AnacondaRecipes/pycodestyle-feedstock/blob/5d1e03d465be51a91b03f079655c46efb853f05e/recipe/meta.yaml#L37

File: recipe/meta.yaml (hunk 2 above)
Comment (Arishamays1) [reply recipe/meta.yaml]: Thanks Serhii!

**Review by varlackc — APPROVED** [review]
LGTM

**Pull request merged**